examples/extract-table-from-image/index.ts

```diff
@@ -0,0 +1,70 @@
+import Instructor from "@/instructor"
+import OpenAI from "openai"
+import { z } from "zod"
+
+const IMAGE_URL = "https://duquark.files.wordpress.com/2019/12/graph.jpg?w=700&h=499"
+
+const TableResponseSchema = z.object({
+  title: z.string(),
+  caption: z.string(),
+  table: z.any()
+})
+type TableResponse = z.infer<typeof TableResponseSchema>
+
+const oai = new OpenAI({
+  apiKey: process.env.OPENAI_API_KEY,
+  organization: process.env.OPENAI_ORG_ID
+})
+
+const instructor = Instructor({
+  client: oai,
+  mode: "MD_JSON"
+})
+
+const extractTableFromImage = async (imageUrl: string): Promise<TableResponse | undefined> => {
+  try {
+    const response = await instructor.chat.completions.create({
+      model: "gpt-4-vision-preview",
+      response_model: {
+        schema: TableResponseSchema,
+        name: "table extraction"
+      },
+      max_tokens: 1_000,
+      messages: [
+        {
+          role: "user",
+          content: [
+            { type: "text", text: "Describe this data accurately as a table in markdown format." },
```

Comment:
the MD_JSON more inserts a system prompt describing the schema and what to output - not sure this is totally necessary - also, this could be conflicting since your telling it to return markdown right?

Reply:
https://github.com/instructor-ai/instructor-js/pull/118/commits/e7cc8c85cc9922ea34f6af8879e93c996285e13d